visualize-admin/visualization-tool
# feat: Add support for showing confidence intervals

Closes #1857

This PR adds support for displaying confidence intervals in Visualize.

**Details**
- Added `ConfidenceUpperBoundDimension` and `ConfidenceLowerBoundDimension`
- Updated Y field type for columns and lines to correctly indicate that it can contain additional, uncertainty-related properties
- Standardized error formatting logic across all charts
- Added a way to display confidence intervals, which use the existing `ErrorWhiskers` components and tooltip formatting
- Fixed showing errors in symbol layer tooltip

## How to test
1. Go to [this link](https://visualization-tool-git-feat-non-symmetrical-error-bar-ixt1.vercel.app/en/create/new?cube=https://mock.ld.admin.ch/cube/mock-two-sided/1&dataSource=Test).
2. ✅ See that confidence intervals are displayed by default.
3. ✅ Open Vertical Axis field and see that there's an option to control their appearance.
4. ✅ Hover over columns and see that the tooltip displays the values in a different way than standard errors.
